### PR TITLE
comma is not needed after URL if you don't use proxy

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9095

### CODE REVIEW

**Comment by illfated** [comment]
( ref. old PR #9228 )

File: windows/security/information-protection/windows-information-protection/create-wip-policy-using-intune-azure.md [hunk 1]
```diff
@@ -629,4 +628,4 @@ You can restrict which files are protected by WIP when they are downloaded from
 - [Azure RMS Documentation Update for May 2016](https://blogs.technet.microsoft.com/enterprisemobility/2016/05/31/azure-rms-documentation-update-for-may-2016/)
 
 >[!NOTE]
->Help to make this topic better by providing us with edits, additions, and feedback. For info about how to contribute to this topic, see [Editing Windows IT professional documentation](https://github.com/Microsoft/windows-itpro-docs/blob/master/CONTRIBUTING.md).
\ No newline at end of file
+>Help to make this topic better by providing us with edits, additions, and feedback. For info about how to contribute to this topic, see [Editing Windows IT professional documentation](https://github.com/Microsoft/windows-itpro-docs/blob/master/CONTRIBUTING.md).
```
Comment (illfated) [inline windows/security/information-protection/windows-information-protection/create-wip-policy-using-intune-azure.md:631]: Please add MarkDown indent marker compatibility spacing.
```suggestion
> Help to make this topic better by providing us with edits, additions, and feedback. For info about how to contribute to this topic, see [Editing Windows IT professional documentation](https://github.com/Microsoft/windows-itpro-docs/blob/master/CONTRIBUTING.md).
```

File: windows/security/information-protection/windows-information-protection/create-wip-policy-using-intune-azure.md [hunk 2]
```diff
@@ -629,4 +628,4 @@ You can restrict which files are protected by WIP when they are downloaded from
 - [Azure RMS Documentation Update for May 2016](https://blogs.technet.microsoft.com/enterprisemobility/2016/05/31/azure-rms-documentation-update-for-may-2016/)
 
 >[!NOTE]
```
Comment (illfated) [inline windows/security/information-protection/windows-information-protection/create-wip-policy-using-intune-azure.md:630]:
```suggestion
> [!NOTE]
```

**Comment by illfated** [comment]
Looks quite OK otherwise.

**Review by illfated — APPROVED** [review]
Looks good to me, I approve of this change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**